sw/airborne/subsystems/ins/ins_int.c

```diff
@@ -54,19 +54,15 @@
 #include "math/pprz_geodetic_int.h"
 #include "math/pprz_isa.h"
 
+#ifndef INS_BARO_AGL_OFFSET
```

Comment:
What is this offset supposed to be?
If it is supposed to be the static offset between the reported AGL and the real AGL, this should not be done in the INS, but the offset directly applied in the module creating the AGL message.